package.json

```diff
@@ -68,7 +68,7 @@
     "prebundle": "yarn i18n-cache:force",
     "prebundle:android": "yarn patch-metro-no-file-watch",
     "postbundle:android": "yarn un-patch-metro-no-file-watch",
-    "bundle": "yarn bundle:android && yarn bundle:ios",
+    "bundle": "yarn bundle:android && yarn bundle:ios && yarn clean:pot && yarn genstrings",
```

Comment:
If there is no "gutenberg*.pot` file, then `yarn clean:pot` fails and prevents `yarn genstrings` from executing.

Reply:
Weird it's not failing for me when there is no pot file, I think this might be because of the wildcard?
I can add `-f` if it fixes it for you 👍

Reply:
Maybe worth moving this to `prebundle` with the rest of the i18n work